mlflow/store/dbmodels/legacy_models.py

```diff
@@ -0,0 +1,235 @@
+# Snapshot of MLflow DB models as of the 0.9.1 release, prior to the first database migration.
+# Used to standardize initial database state.
```

Comment:
In particular, this file is used to create initial database state before running migrations, so that we don't need an "init" migration that sets up the database schema before the introduction of alembic/DB migrations. Writing such an "init" migration is tricky with Alembic APIs (i.e. there isn't a nice API for statements like "CREATE TABLE IF NOT EXISTS..."). Moreover, using a Python file to create initial database state ensures that there's only a single DB initialization/migration logic, including for users running SQL-backed stores prior to this PR.